0000-JWT-Code.md

```diff
@@ -2134,8 +2134,9 @@ public class JWTWebSecurityConfig extends WebSecurityConfigurerAdapter {
     @Value("${jwt.get.token.uri}")
     private String authenticationPath;
 
-    @Autowired
-    public void configureGlobal(AuthenticationManagerBuilder auth) throws Exception {
+    //@Autowired why ?? and why configureGlobal ??
+    @Override
```

Comment:
+1 to this. Independently found the same solution. It seems like this should be `@Override` and not `@Autowired`.

Reply:
Doing the above change gets ride of this error: 
```
org.springframework.beans.factory.BeanCurrentlyInCreationException: Error creating bean with name 'JWTWebSecurityConfig': Requested bean is currently in creation: Is there an unresolvable circular reference?
```